osu.Game/Graphics/Containers/Markdown/OsuMarkdownHeading.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Syntax;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Framework.Graphics.Sprites;
+
+namespace osu.Game.Graphics.Containers.Markdown
+{
+    public class OsuMarkdownHeading : MarkdownHeading
+    {
+        private readonly int level;
+
+        public OsuMarkdownHeading(HeadingBlock headingBlock)
+            : base(headingBlock)
+        {
+            level = headingBlock.Level;
+        }
+
+        public override MarkdownTextFlowContainer CreateTextFlow() => new HeadingTextFlowContainer
+        {
+            Weight = GetFontWeightByLevel(level),
+        };
+
+        protected override float GetFontSizeByLevel(int level)
+        {
+            const float base_font_size = 14;
```

Comment:
Worth including an inline comment referencing where these sizes come from (ie. a link to the osu-web source)